tensorboard/backend/event_processing/event_accumulator.py

```diff
@@ -455,6 +455,10 @@ def Graph(self):
       return graph
     raise ValueError('There is no graph in this EventAccumulator')
 
+  def SerializedGraph(self):
```

Comment:
+1 to avoid changing the non-plugin event accumulator, since it's basically only maintained for legacy purposes, and we should avoid suggesting that it's still getting new features.